.storybook/babel.config.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         },
       },
     ],
-    '@babel/preset-react',
+    ['@babel/preset-react', { runtime: 'automatic' }],
```

Comment:
Small thing, could you add a comment above this just to clarify why we have this turned on just for future maintenance